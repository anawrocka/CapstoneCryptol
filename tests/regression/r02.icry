[1 .. 10] + [10,9 .. 1]
[1 .. 10] + [10,9 .. 1] == [11, 11, 11, 11, 11, 11, 11, 11, 11, 11]
