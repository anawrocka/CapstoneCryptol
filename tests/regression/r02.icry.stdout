Loading module Cryptol
Assuming a = 4
[0xb, 0xb, 0xb, 0xb, 0xb, 0xb, 0xb, 0xb, 0xb, 0xb]
[warning] at <interactive>:1:1--1:68:
  Defaulting type parameter 'bits'
             of literal or demoted expression
             at <interactive>:1:65--1:67
  to 4
True
